app/views/comments/_comments.html.slim

```diff
@@ -0,0 +1,10 @@
+- content_for :javascript do
```

Comment:
sometimes yes, but this JS is small and you can simply include it in application.js

by inialize I mean something like

```
javascript:
$(ready() {  initCommentsPage(); })
```

basically this function is equal to what you have in $.ready now in your JS